data/db-schema/src/main/resources/net/corda/db/schema/statemanager/migration/state-manager-migration-v1.0.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <property name="now" value="now()" dbms="postgresql"/>
+    <property name="json.column.type" value="JSONB" dbms="postgresql"/>
+    <property name="state.column.type" value="bytea" dbms="postgresql"/>
+
+    <!-- Please note that HSQLDB schema is supported for integration test purposes only. -->
+    <property name="now" value="now()" dbms="hsqldb"/>
+    <property name="json.column.type" value="CLOB" dbms="hsqldb"/>
+    <property name="state.column.type" value="binary" dbms="hsqldb"/>
+
+    <changeSet author="R3.Corda" id="state-manager-migration-v1.0">
+        <createTable tableName="state">
+            <column name="key" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="state" type="${state.column.type}">
+                <constraints nullable="false"/>
```

Comment:
May need to double check this, but when we complete a flow, we null the state, we'll have to now delete the key  from state storage if we can't null the state (probably a better thing anyway)